doc/transformation.md

```diff
@@ -0,0 +1,24 @@
+#Transformation Writeup
+
+I wonder what this really is... [enc](https://mercury.picoctf.net/static/1d8a5a2779c4dc24999f0358d7a1a786/enc) \
+`''.join([chr((ord(flag[i]) << 8) + ord(flag[i + 1])) for i in range(0, len(flag), 2)])`
+
+Downloading and opening the enc file we see "灩捯䍔䙻ㄶ形楴獟楮獴㌴摟潦弸彥㜰㍢㐸㙽". The name of the file is enc so this is
+probably the encrypted flag. Looking back at the problem, we can see a little code snippet. The variable name that was
```

Comment:
"The variable name that was used was flag" is a run-on sentence. --> There is a variable named as `flag` in the code.

Reply:
it is not a run on, but fixed